src/Razor/test/Microsoft.VisualStudio.LanguageServices.Razor.Test/LspEditorFeatureDetectorTest.cs

```diff
@@ -18,21 +18,19 @@ namespace Microsoft.VisualStudio.Razor;
 
 public class LspEditorFeatureDetectorTest(ITestOutputHelper testOutput) : ToolingTestBase(testOutput)
 {
-    public static TheoryData<bool, bool, bool> IsLspEditorEnabledTestData { get; } = new()
+    public static TheoryData< bool, bool> IsLspEditorEnabledTestData { get; } = new()
```

Comment:
Nit:
```suggestion
    public static TheoryData<bool, bool> IsLspEditorEnabledTestData { get; } = new()
```

I would have hoped running Format Document would have fixed that by itself